book/workflow.rst

```diff
@@ -0,0 +1,236 @@
+Workflows and Processes
+=======================
+
+In a business application, entities often have a well defined state and undergo transitions between
+different states. The OroPlatform comes with two solutions to apply such a system to your entities:
+
+* `Workflows`_ are a way to apply states and transitions that are triggered by user actions through
+  the UI.
+
+* If you want to automate some steps, you can use `Processes`_ which are performed in the
```

Comment:
Well, processes are not the tool to automate transitions - they can be used to do that, but their original purpose was implementation of some optional actions that can be performed after some specific trigger actions. In fact, processes can be considered as slightly improved Doctrine event listeners, and improvements are: optional usage (can be disabled), asynchronous execution (via job queue), definition in declaration style (YAML).